### PR TITLE
New arch: Solved jetson problem. memory leak, and changed the architecture a little

### PR DESCRIPTION
**Pull request by JsonSadler**
Hi, 

I solved the problem with jetson - cuda context and related stuff was messed up, needed architecture to be changed a little bit to solve (Thread related stuff). There was also some problems with freeing pycuda driver in the correct moment to let objects be destroyed in the right place (It caused some memory leaks that are fixed now). I have tested all the docker files except for coral-dev-board. 
I also removed RESTART from api and commands in core, as now the engine is built only when needed, while processing the video. I have not tested API's set for changing the video path but it should work. 

Let me know if there was any problems.

### CODE REVIEW

File: jetson-run-frontend.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM nvcr.io/nvidia/l4t-base:r32.3.1
```
Comment (JsonSadler) [inline jetson-run-frontend.Dockerfile:1]: I also added jetson-run-frontend.Dockerfile  for ARM, if any one wanted to run the Dashboard on jetson too.

File: libs/detectors/jetson/detector.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
+
+import logging
```
Comment (JsonSadler) [inline libs/detectors/jetson/detector.py:2]: Oops, these imports are not necessary, are residuals of my debugging process, you can remove them in your PR

File: jetson-run-frontend.Dockerfile (hunk 1 above)
Comment (undefined-references) [reply jetson-run-frontend.Dockerfile:1]: thanks

**Comment by undefined-references** [comment]
Thanks for your PR @JsonSadler

**Pull request merged**